src/utils/compat.c

```diff
@@ -12,3 +12,10 @@ char *strdup(const char *s) {
     return d;
 }
 #endif
+
+#ifndef _WIN32
```

Comment:
Should we let cmake do this check, just like HAVE_STD_STRDUP ?

Reply:
No, I mean drop the define check and rename the function omf_strnlen_s or somesuch.

And yes, If possible, we should keep the platform-specific define-checks to minimum. I realize it's not possible to remove them all, but let's at least try. Otherwise we are going to end up macro-soup.

Reply:
Sounds good, I'll provide it as omf_strnlen_s in c_string_util.h